README.md

```diff
@@ -11,34 +11,44 @@
 **[Explore our docs](https://docs.unskript.com)**   
 *[Visit our blog](https://unskript.com/blog)* . *[Report Bug](https://github.com/unskript/Awesome-CloudOps-Automation/issues/new?assignees=&labels=&template=bug_report.md&title=)* . *[Request Feature](https://github.com/unskript/Awesome-CloudOps-Automation/issues/new?assignees=&labels=&template=feature_request.md&title=)*
 
-# About the Project
-Runbooks.sh is a powerful, community-driven, open-source runbook automation platform designed to simplify cloud infrastructure management and streamline operations across diverse environments. Few of the highlighting features:
+# 🚀 Quick Start Guide
 
-- **Extensive Library**: Access hundreds of pre-built actions and runbooks to kickstart your automation journey.
-- **Customization**: Create and modify actions and runbooks tailored to your unique requirements.
-- **Generative AI Action Creation** Fully integrated with ChatGPT to create custom Actions in minutes.
-- **Diverse Compatibility**: Seamlessly integrate with various cloud providers, platforms, and tools.
-- **User-friendly Interface**: A Jupyter-based environment that simplifies runbook creation and execution.
-- **Active Community**: Join a vibrant community of users and contributors committed to improving the project.
+We recommend using our docker setup which comes with Jupyter runtime along with pre-built [actions](https://docs.unskript.com/unskript-product-documentation/actions/what-is-an-action) and [runbooks](https://docs.unskript.com/unskript-product-documentation/readme/what-is-a-runbook). Build your own actions and runbooks with ease!
 
-## 🏆 Mission
-Our mission is to simplify CloudOps automation for DevOps and SRE teams by providing an extensive, community-driven repository of actions and runbooks that streamline day-to-day operations. 
+### Get Started
 
-## 👁️ Vision 
-Our vision is to be the one-stop solution for all CloudOps automation needs, allowing DevOps and SRE teams to automate their workflows with ease, improve efficiency, and minimize toil.
+1. Launch Docker
+```
+docker run -it -p 8888:8888 --user root unskript/awesome-runbooks:latest
+```
+2. Point your browser to http://127.0.0.1:8888/awesome.
 
-## 🚀 Quick Start Guide
 
-We recommend using our docker setup which comes with Jupyter runtime along with pre-built actions and runbooks. Build your own actions and runbooks with ease!
 
-### Get Started
+### Messing around
```

Comment:
I think we should name this section `Advanced Usage` or `Adding Custom Actions/Runbooks`
The two use cases enabled are 

1. Custom Action and Runbook creation
2. OpenAI integration

Messing around seems very non-specific :-)